Misc/NEWS.d/next/Library/2024-10-18-09-51-29.gh-issue-125682.vsj4cU.rst

```diff
@@ -0,0 +1,2 @@
+Reject non-ASCII digits in the Python implementation of :func:`json.loads`
+conforming to the json specification.
```

Comment:
Nitpick:
```suggestion
conforming to the JSON specification.
```

Reply:
Yeah, it became lowercase when I translated the sentence with DeepL.